src/app/modules/+sign-in/sign-in/sign-in.component.ts

```diff
@@ -21,4 +26,11 @@ export class SignInComponent implements OnInit {
       password: ['', Validators.required]
     });
   }
+
+  onSubmit(): void {
+    this.authUserService.authenticate(this.signInForm.value)
+    .subscribe(() => {
+        this.router.navigate(['/dashboard']);
+      });
+  }
```

Comment:
I would recommend always check if form is valid if you make some request based on form values (disabled button can be easily enabled from browser dev tools). You will need to make following changes:
 
if (this.signInForm.valid) {
   this.authService.authenticate(this.signInForm.value).subscribe(() => this.router.navigate(['/dashboard']));
}